src/utils/vacation-days-utils.tsx

```diff
@@ -0,0 +1,43 @@
+import { Typography, Box} from "@mui/material";
+import { theme } from "../theme";
+import { Person } from "../generated/client";
+import strings from "../localization/strings";
+import { useState } from "react";
+import { personsAtom } from "../atoms/person";
+import { useAtom } from "jotai";
+
+/**
+ * Display persons vacation days
+ * @param Person timebank person
+ */
+export const renderVacationDays = (person: Person) => {
+const loading = useState(true);
+const persons = useAtom(personsAtom);
+
+const spentVacationsColor =
+    person.spentVacations > 0
+        ? theme.palette.success.main
+        : theme.palette.error.main;
+
+const unspentVacationsColor =
+    person.unspentVacations > 0
+        ? theme.palette.success.main
+        : theme.palette.error.main;
+
+if (!loading && persons.length) {
+    return <Typography>{strings.error.fetchFailedNoEntriesGeneral}</Typography>;
```

Comment:
Is this the correct error message to use here, wouldn't the error be related to persons rather than just a general no entries?